src/Injections/ICameraHandler.cs

```diff
@@ -4,40 +4,50 @@
 using CSM.Commands.Data.Internal;
 using CSM.Panels;
 using HarmonyLib;
+using System;
 
 namespace CSM.Injections
 {
     [HarmonyPatch(typeof(CameraController))]
-    [HarmonyPatch("UpdateTransform")]
+    [HarmonyPatch("LateUpdate")]
     public class ICameraUpdateCurrentPosition
     {
         private static Vector3 playerCameraPosition_last;
         private static Quaternion playerCameraRotation_last;
 
         public static void Postfix(CameraController __instance, Camera ___m_camera)
         {
-            // Get camera rotation, angle and position
-            Transform transform = ___m_camera.transform;
-            Quaternion _rotation = transform.rotation;
-            Vector3 _position = transform.position;
-
-            // Check if the camera moved
-            if (Vector3.Distance(_position, playerCameraPosition_last) > 1 || playerCameraRotation_last != _rotation)
+            // Make sure the camera object is defined, this should always happen but for some strange reason it didnt (look bellow)
+            // This is not happening if i test it localy. (debug using AllocConsole in CSM.cs)
+            // https://github.com/DominicMaas/Tango/issues/155
+            if (___m_camera && MultiplayerManager.Instance.PlayerList.Count > 0)
```

Comment:
Please use `MultiplayerManager.Instance.CurrentRole != MultiplayerRole.None` instead of the player list check.

Reply:
I have applied the changes you have requested